resources/styles/book-content/context-mixins.less

```diff
@@ -0,0 +1,17 @@
+.tutor-book-note(reading) {
+  > .note,
+  > section > .note:not(.learning-objectives):not(.tips-for-success),
+  section > section > .note {
+    background: none;
+    padding: 20px 0;
+    &:before {
+      margin: -68px 0 0 0;
```

Comment:
Where did this margin calc come from? Is there a var (or var + number) you could use in case we change margins?